glean-core/android/src/main/java/mozilla/telemetry/glean/Glean.kt

```diff
@@ -102,15 +102,19 @@ open class GleanInternalAPI internal constructor () {
      * @param applicationContext [Context] to access application features, such
      * as shared preferences
      * @param configuration A Glean [Configuration] object with global settings.
+     * @param uploadEnabled A [Boolean] that determines the initial state of the uploader
      */
     @Suppress("ReturnCount", "LongMethod", "ComplexMethod")
     @JvmOverloads
     @Synchronized
     @MainThread
     fun initialize(
         applicationContext: Context,
-        configuration: Configuration = Configuration()
+        configuration: Configuration = Configuration(),
+        uploadEnabled: Boolean
     ) {
+        this.uploadEnabled = uploadEnabled
```

Comment:
I know the behaviour is basically the same, but can we please call `setUploadEnabled(uploadEnabled)` here instead?